tests/test_cl2hpp.cpp

```diff
@@ -2484,8 +2523,13 @@ void testPlatformExtendedVersioning()
     TEST_ASSERT_EQUAL(extensions.size(), 1);
     TEST_ASSERT_EQUAL_HEX(extensions[0].version, CL_MAKE_VERSION_KHR(10, 11, 12));
     TEST_ASSERT_EQUAL_STRING(extensions[0].name, "cl_dummy_extension");
+#endif // CL_HPP_TARGET_OPENCL_VERSION < 300
```

Comment:
I think I'd rather we statically assert that the enums have the same value and use one rather than duplicate the test code.

Reply:
I don't have a very strong opinion about this.  At least for the device queries, the KHR version tests `CL_DEVICE_OPENCL_C_NUMERIC_VERSION_KHR` and the 3.0 version doesn't.  Are you concerned about maintainability?  Test execution time?  Something else?

I've left this as-is for now.